lib/sdf-server/src/server/service/v2/variant.rs

```diff
@@ -100,7 +162,11 @@ pub async fn get_variant(
 
     let schema_variant = SchemaVariant::get_by_id_or_error(&ctx, schema_variant_id).await?;
     let schema_id = SchemaVariant::schema_id_for_schema_variant_id(&ctx, schema_variant_id).await?;
-    let schema_variant = schema_variant.into_frontend_type(&ctx, schema_id).await?;
+    let mut schema_variant = schema_variant.into_frontend_type(&ctx, schema_id).await?;
+
+    if let Err(err) = determine_can_update_one(&ctx, &mut schema_variant, &raw_access_token).await {
```

Comment:
We don't use this route ever (and normally, I'd delete dead code), but I covered it off anyway since there's active work here. 
cc @jobelenus @vbustamante @wendybujalski